tdei-ui/src/components/CreateProjectGroup/CreateProjectGroup.js

```diff
@@ -102,6 +102,7 @@ const CreateProjectGroup = (props) => {
                     onBlur={handleBlur}
                     value={values.project_group_name}
                     isInvalid={touched.project_group_name && !!errors.project_group_name}
+                    disabled={project_group_name === "Default"}
```

Comment:
Make constant of "Default"  and use in application.

Reply:
resolved